python-lib/sharepoint_client.py

```diff
@@ -87,6 +87,11 @@ def __init__(self, config):
                 username=username,
                 password=password
             )
+            self.form_digest_value = get_form_digest_value(self.sharepoint_url, self.sharepoint_site, session=self.session)
+            default_headers = None
```

Comment:
🧑‍🍳 suggestion:
```suggestion
            default_headers = {"X-RequestDigest": self.form_digest_value} if self.form_digest_value else None
```

Reply:
Done in [b21dd28](https://github.com/dataiku/dss-plugin-sharepoint-online/pull/53/commits/b21dd28692865e6cd3a8d56c66849ffa347e8d65)